lnwallet/interface.go

```diff
@@ -98,6 +98,15 @@ type OutputDetail struct {
 	IsOurAddress bool
 }
 
+// PreviousOutPoint contains information about the previous outpoint.
+type PreviousOutPoint struct {
+	// The outpoint in format txid:n.
```

Comment:
Godoc comments (any comment above a type, method, function or struct field) must start with the type name. So this should read `OutPoint is the transaction out point in the format txid:n.`, same with the `IsOurOutput` below.